### ISSUE TITLE
Feature Request: 

### ISSUE THREAD

**Issue by MateiNenciu** [chronotruck/vue-ctk-date-time-picker#30]
If you could implement text box feature like in elementui DateTimePicker it would be one of the best DT pickers for VueJs.

Example: https://element.eleme.io/#/en-US/component/datetime-picker

Thanks !

**Comment by mankey-ru** [comment]
Yes, Im migrating from old legacy code with jQuery UI Datepicker and this is the only feature missing. Thank you very much for the best Vue datepicker!
